src/Microsoft.Performance.SDK/Processing/DataSourceAttribute.cs

```diff
@@ -93,8 +93,8 @@ protected DataSourceAttribute(Type type, string description)
 
         /// <summary>
         ///     This method is used to filter anyincoming Data Sources so that only
-        ///     Data Sources that could conceivably be processed by the decorated
-        ///     Custom Data Source are passed to the Custom Data Source for evaluation.
+        ///     <see cref="IDataSource"/> that could conceivably be processed by the decorated
```

Comment:
> <see cref="IDataSource"/> that could conceivably [](http://example.com/codeflow?start=15&length=49)

do we need an article in front of IDataSource? or make it plural

Reply:
I think plural makes sense here.